### PR TITLE
USDC/USDCe Curve strat

### PR DESCRIPTION
**Pull request by 0xKirkjufell**
If you made a contract change, make sure to complete the checklist below before merging it in master.

Contract change checklist:
  - [x] Code reviewed by 1 reviewers.
  - [x] Unit tests pass – `yarn test`
  - [x] Smoke tests pass: `./scripts/smokeTest.sh`
  - [x] Slither tests pass with no warning: `yarn slither` (repair with `yarn slither:triage`)
  - [x] Echidna tests pass if PR includes changes to XUSD contract (not automated, run manually on local): `yarn echidna`

### CODE REVIEW

File: contracts/mocks/curve/MockCurveGauge.sol [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    }
+
+    function setRewardAmount(uint256 _amount) external {
+        amount = _amount;
+    }
+
+    function claim_rewards(address _sender, address _receiver) external override {
+        address reward = reward_tokens[0];
+        if (amount > 0) {
+            // TODO: loop over multiple rewards
```
Comment (0xKirkjufell) [inline contracts/mocks/curve/MockCurveGauge.sol:46]: also, add test case for multiple rewards, e.g. WAVAX, CRV

File: contracts/oracle/OracleRouter.sol [hunk 2]
```diff
@@ -57,13 +57,17 @@ contract OracleRouter is OracleRouterBase {
             return address(0x51D7180edA2260cc4F6e4EebB82FEF5c3c2B8300);
         } else if (
             // USDCe
-            asset == address(0xA7D7079b0FEaD91F3e65f86E8915Cb59c1a4C664)
+            asset == address(0xA7D7079b0FEaD91F3e65f86E8915Cb59c1a4C664) ||
```
Comment (0xKirkjufell) [inline contracts/oracle/OracleRouter.sol:60]: Open to discussion of the right design choice here as USDC and USCDe would be equivalent (even though they are not )

File: utils/constants.js [hunk 3]
```diff
@@ -1 +1,3 @@
-module.exports = {};
+const numberOfStrategies = 2;
```
Comment (0xKirkjufell) [inline utils/constants.js:1]: remove this, unused

File: test/abi/crvRewarder.json [hunk 4]
```diff
@@ -0,0 +1 @@
...outputs":[],"gas":37700},{"stateMutability":"view","type":"function","name":"owner","inputs":[],"outputs":[{"name":"","type":"address"}],"gas":2658},{"stateMutability":"view","type":"function","name":"future_owner","inputs":[],"outputs":[{"name":"","type":"address"}],"gas":2688},{"stateMutability":"view","type":"function","name":"reward_receiver","inputs":[],"outputs":[{"name":"","type":"address"}],"gas":2718},{"stateMutability":"view","type":"function","name":"reward_tokens","inputs":[{"name":"arg0","type":"uint256"}],"outputs":[{"name":"","type":"address"}],"gas":2857},{"stateMutability":"view","type":"function","name":"reward_count","inputs":[],"outputs":[{"name":"","type":"uint256"}],"gas":2778},{"stateMutability":"view","type":"function","name":"reward_data","inputs":[{"name":"arg0","type":"address"}],"outputs":[{"name":"distributor","type":"address"},{"name":"period_finish","type":"uint256"},{"name":"rate","type":"uint256"},{"name":"duration","type":"uint256"},{"name":"received","type":"uint256"},{"name":"paid","type":"uint256"}],"gas":14685},{"stateMutability":"view","type":"function","name":"last_update_time","inputs":[],"outputs":[{"name":"","type":"uint256"}],"gas":2838}]
```
Comment (0xKirkjufell) [inline test/abi/crvRewarder.json:1]: remove

**Review by 0xAnnapurna — APPROVED** [review]

**Pull request merged**